resources/public/api/conf/1.0/relief_investments_delete.yaml

```diff
@@ -16,12 +16,16 @@ description: |
       <tbody>
           <tr>
               <td><p>N/A - DEFAULT</p></td>
-              <td><p>Simulate a success response.</p></td>
+              <td><p>Simulates a success response.</p></td>
           </tr>
           <tr>
               <td><p>NOT_FOUND</p></td>
               <td><p>Simulates the scenario where no data is found.</p></td>
-           </tr>
+          </tr>
+          <tr>
+              <td><p>STATEFUL</p></td>
+              <td><p>Simulates a stateful success response.</p></td>
```

Comment:
Relief investments:
the stateful description needs to be updated to the newer version!